src/util/logIfForbidden.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  IntegrationLogger,
+  IntegrationError,
+} from '@jupiterone/integration-sdk-core';
+
+class IntegrationApiError extends IntegrationError {
```

Comment:
There is an error class of a similar purpose in the SDK. I wonder if it can be used?